samples/net/common/common.cmake

```diff
@@ -1,3 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
I like the snippet commit, there is probably other generic parts we can put to snippets in the future, at least the VLAN config is one candidate.